pandas/tests/indexes/timedeltas/test_tools.py

```diff
@@ -37,10 +40,11 @@ def conv(v):
         expected = pd.Index(np.array([np.timedelta64(1, 's')]))
         tm.assert_index_equal(result, expected)
 
-        # ints
-        result = np.timedelta64(0, 'ns')
-        expected = to_timedelta(0, box=False)
-        assert result == expected
+        with tm.assert_produces_warning(FutureWarning):
+            # ints
```

Comment:
so if box is not relevant, then ok with simply changing the tests (not specific to here, just generally)

Reply:
You mean ok to remove tests for `box=False`?

Reply:
you can remove a good bit of them, ideally leave *some* of them, just don't need all.

Reply:
Ok. The heuristic I used was to leave box for the core to_timedelta and to_timestamp tests so that the box functionality is still being tested, and of course in the tests that are testing for the deprecation.  If box was just incidental to the test, then changed it to not use the box argument.